pkg/model/config/config.go

```diff
@@ -104,6 +93,25 @@ func (config *Config) AddResource(gvk GVK) bool {
 	return true
 }
 
+// HasGroup returns true if group is already tracked
+// NOTE: in v1 resources are not tracked, so we return false
+func (config Config) HasGroup(group string) bool {
+	// Short-circuit v1
+	if config.IsV1() {
+		return false
+	}
```

Comment:
just remove it .. if HasGroup should not check if isV1 or V2. 
if isV1 the config.Resources will be empty and it will return false.

Reply:
From the method comment:
> NOTE: in v1 resources are not tracked, so we return false
Checking version maintains legacy behavior.

Reply:
hi @estroz,

The ` legacy behaviour` was check in the command the version in order to verify if is MultiGroup enabled or not since it is a feature required only for V2(V1 allows multigroup by default). Then, in POV a new method `HasGroup` in the config which can be used in any place not requires it.

Reply:
Hi @estroz,

Really tks for the input.

However, note that the legacy behaviour was; when the api command is performed then check if is multigroup supported or not (v2 do not support multigroup by default)

I do not think that HasGroup is reasonable for check versions, and also it is not required at all.

Reply:
Both `HasGroup` and `HasResource` had shortcircuits for v1, but as v1 doesn't track resources the behavior without the shortcircuits was the same, so I guess we can remove them.